version.gradle

```diff
@@ -1,3 +1,3 @@
 ext {
-  PLUGIN_VERSION = '0.5.0-20'
+  PLUGIN_VERSION = '0.6.0-1'
```

Comment:
Adding another feature still doesn't change the target version under which we're gonna release it: it's gonna be 0.5.0 regardless of whether there's 1 or 10 new features added.

Note that would be different if the version up to this PR was e.g. `0.5.1-20`; this would mean that the planned release (0.5.1) so far was going to contain only bugfixes/internal changes. In such case, adding a new feature would indeed make a difference: we'd need to change the number of the planned release to `0.6.0`; `0.5.1` would hence never get released as such.

Reply:
So... in short, pls change to `0.5.0-21` ;)